src/components/UppyDashboard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import '@uppy/core/dist/style.css';
+import '@uppy/dashboard/dist/style.css';
+
+import { Dashboard } from '@uppy/react';
+import { useTranslation } from 'react-i18next';
+
+interface FileUploaderProps {
+  uppy: any;
+  maxWidthPx?: number;
+  maxHeightPx?: number;
+}
+
+const UppyDashboard: React.FC<FileUploaderProps> = ({ uppy, maxWidthPx = 2000, maxHeightPx = 200 }) => {
+  const { t } = useTranslation('publication');
+
+  return uppy ? (
+    <Dashboard
+      uppy={uppy}
+      proudlyDisplayPoweredByUppy={false}
+      showSelectedFiles={false}
+      showProgressDetails
+      hideProgressAfterFinish
+      width={maxWidthPx}
```

Comment:
TODO: prøv å få til boxing av dette. Virker for files tab, men ikke startside atm